README.md

```diff
@@ -93,6 +93,10 @@ After a successful build, you should have a `glslc` executable somewhere under
 the `$BUILD_DIR/glslc/` directory, as well as a `libshaderc` library somewhere
 under the `$BUILD_DIR/libshaderc/` directory.
 
+The default behavior on MSVC is to link with the static CRT. If you would like
```

Comment:
I just wasn't sure that "static CRT" is synonymous with "CRT statically linked into an executable/library/DLL".  But it [sounds](https://msdn.microsoft.com/en-us/library/abx4dbyh(v=vs.120).aspx) like it is.